modules/Bio/EnsEMBL/DBSQL/SOTermAdaptor.pm

```diff
@@ -1,77 +0,0 @@
-=head1 LICENSE
-
-Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
-Copyright [2016-2019] EMBL-European Bioinformatics Institute
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-     http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
-=cut
-
-
-=head1 CONTACT
-
-  Please email comments or questions to the public Ensembl
-  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
-
-  Questions may also be sent to the Ensembl help desk at
-  <http://www.ensembl.org/Help/Contact>.
-
-=cut
-
-=head1 NAME
-
-Bio::EnsEMBL::DBSQL::SOTermAdaptor
-
-=head1 DESCRIPTION
-
-A specialization of Bio::EnsEMBL::DBSQL::OntologyTermAdaptor,
-specifically for Sequence ontology (SO) terms.  See the
-documentation of Bio::EnsEMBL::DBSQL::OntologyTermAdaptor for
-further information.
-
-=head1 METHODS
-
-=cut
-
-package Bio::EnsEMBL::DBSQL::SOTermAdaptor;
```

Comment:
There's a reference to SOTermAdaptor in ensembl-variation. However, I'm not seeing that method ever gets called anywhere. For that matter, I'm not sure that method still works (for example, where does $self->{adaptor} ever get set in that class as currently implemented?). Have we gotten an OK or don't care from Variation on this?

https://github.com/Ensembl/ensembl-variation/blob/396640b5e0f5e03f2ddfa863948f291a1e59eb07/modules/Bio/EnsEMBL/Variation/OverlapConsequence.pm#L493

Reply:
> Have we gotten an OK or don't care from Variation on this?

I would say we don't care, seeing as technically speaking that call should have been non-functional since March 2018. Still, I've just pinged them about it on Slack - just in case.

Reply:
Variation have just confirmed that the function in question is no longer in use (and they'll remove it now).